config/deploy.rb

```diff
@@ -15,6 +15,8 @@
 
 set :branch, 'master'
 
+set :passenger_restart_with_touch, true
```

Comment:
Nope, it should all just work. I think we discovered `passenger_restart_with_touch` is unnecessary too, right?